src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -59,8 +59,8 @@ public FilterCommand parse(String args) throws ParseException {
             String substring = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()).value;
             return new FilterCommand(new AddressContainsSubstringPredicate(substring));
         }
-        if (argMultimap.getValue(PREFIX_REMARK).isPresent()) {
-            String substring = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_REMARK).get()).value;
+        if (argMultimap.getValue(PREFIX_NEW_REMARK).isPresent()) {
+            String substring = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_NEW_REMARK).get()).value;
```

Comment:
Filtering by the flag `rn/` instead of `r/` may result in user confusion, as `rn/` does not stand for `remark`, but `remark new` instead. Users may prefer to use `r/` as it is more intuitive for use with `filter`.

I understand your intention with the replacement though, as having a growing number of flags could make it harder for users to remember or differentiate between them.

Perhaps we could have internal discussion about this!